compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -2677,7 +2677,10 @@ object Types {
       else {
         if (isType) {
           val res =
-            if (currentSymbol.isAllOf(ClassTypeParam)) argForParam(prefix)
+            val sym =
+              if (currentSymbol.isValidInCurrentRun) currentSymbol
+              else computeSymbol
```

Comment:
Looks like both cases can use `stableInRunSymbol`.

Reply:
Yes, looks like this indeed! Too late for this PR, but could you make another one with the change?